src/clib/initialize_metal_chemistry_rates.cpp

```diff
@@ -248,141 +190,134 @@ int grackle::impl::initialize_metal_chemistry_rates(
-        my_rates->kz16[i] = fmax(my_rates->kz16[i], tiny) / kunit;
-        my_rates->kz17[i] = fmax(my_rates->kz17[i], tiny) / kunit;
-        my_rates->kz18[i] = fmax(my_rates->kz18[i], tiny) / kunit;
-        my_rates->kz19[i] = fmax(my_rates->kz19[i], tiny) / kunit;
-        my_rates->kz20[i] = fmax(my_rates->kz20[i], tiny) / kunit;
-        my_rates->kz21[i] = fmax(my_rates->kz21[i], tiny) / kunit;
-        my_rates->kz22[i] = fmax(my_rates->kz22[i], tiny) / kunit;
-        my_rates->kz23[i] = fmax(my_rates->kz23[i], tiny) / kunit;
-        my_rates->kz24[i] = fmax(my_rates->kz24[i], tiny) / kunit;
-        my_rates->kz25[i] = fmax(my_rates->kz25[i], tiny) / kunit;
-        my_rates->kz26[i] = fmax(my_rates->kz26[i], tiny) / kunit;
-        my_rates->kz27[i] = fmax(my_rates->kz27[i], tiny) / kunit;
-        my_rates->kz28[i] = fmax(my_rates->kz28[i], tiny) / kunit;
-        my_rates->kz29[i] = fmax(my_rates->kz29[i], tiny) / kunit;
-        my_rates->kz30[i] = fmax(my_rates->kz30[i], tiny) / kunit;
-        my_rates->kz31[i] = fmax(my_rates->kz31[i], tiny) / kunit;
-        my_rates->kz32[i] = fmax(my_rates->kz32[i], tiny) / kunit;
-        my_rates->kz33[i] = fmax(my_rates->kz33[i], tiny) / kunit;
-        my_rates->kz34[i] = fmax(my_rates->kz34[i], tiny) / kunit;
-        my_rates->kz35[i] = fmax(my_rates->kz35[i], tiny) / kunit;
-        my_rates->kz36[i] = fmax(my_rates->kz36[i], tiny) / kunit;
-        my_rates->kz37[i] = fmax(my_rates->kz37[i], tiny) / kunit;
-        my_rates->kz38[i] = fmax(my_rates->kz38[i], tiny) / kunit;
-        my_rates->kz39[i] = fmax(my_rates->kz39[i], tiny) / kunit;
-        my_rates->kz40[i] = fmax(my_rates->kz40[i], tiny) / kunit;
-        my_rates->kz41[i] = fmax(my_rates->kz41[i], tiny) / kunit;
-        my_rates->kz42[i] = fmax(my_rates->kz42[i], tiny) / kunit;
-        my_rates->kz43[i] = fmax(my_rates->kz43[i], tiny) / kunit;
-        my_rates->kz44[i] = fmax(my_rates->kz44[i], tiny) / kunit;
-        my_rates->kz45[i] = fmax(my_rates->kz45[i], tiny) / kunit;
-        my_rates->kz46[i] = fmax(my_rates->kz46[i], tiny) / kunit;
-        my_rates->kz47[i] = fmax(my_rates->kz47[i], tiny) / kunit;
-        my_rates->kz48[i] = fmax(my_rates->kz48[i], tiny) / kunit;
-        my_rates->kz49[i] = fmax(my_rates->kz49[i], tiny) / kunit;
-        my_rates->kz50[i] = fmax(my_rates->kz50[i], tiny) / kunit;
-        my_rates->kz51[i] = fmax(my_rates->kz51[i], tiny) / kunit;
-        my_rates->kz52[i] = fmax(my_rates->kz52[i], tiny) / kunit;
-        my_rates->kz53[i] = fmax(my_rates->kz53[i], tiny) / kunit;
-        my_rates->kz54[i] = fmax(my_rates->kz54[i], tiny) / kunit;
+  // at this point, all values in kcol_rate_tables initialized by this
+  // function have cgs units. We need to convert to code units & enforce a
+  // minimum value
+  //
+  // TODO: in the future, we should refactor to make this logic look more
+  //       consistent with the logic for other similar rates in
+  //       initialize_rates
```

Comment:
Maybe this answers my previous question about standardizing with the other rates.